shared/auth/auth-mgr.go

```diff
@@ -0,0 +1,128 @@
+package auth
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+)
+
+const (
+	AuthorizationHeader string = "Authorization"
+	BearerPrefix        string = "Bearer "
+
+	// How long a JWT is valid for after sending a request
+	ValidTime time.Duration = time.Second * 5
```

Comment:
what is * 5 here?

Reply:
It means 5 seconds, that's how you do const time durations. Basically when the daemons / CLI issue a request to another daemon, that request has to make it to the remote end within 5 seconds for it to be valid. Since everything's on the same machine it's basically immediate and this is kind of a formality anyway.